### PR TITLE
v2v: update case env_leak according the changes in libosinfo

### PR DESCRIPTION
**Pull request by xiaodwan**
see: bz#2051559

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.env_leak.esx_67.json.default.it_vddk.vpx_uri: PASS (376.54 s)

File: v2v/tests/cfg/function_test_esx.cfg [hunk 1]
```diff
@@ -654,4 +653,4 @@
             status_error = 'no'
         - negative_test:
             status_error = 'yes'
-            only option_root.single,invalid_rhv_pem,no-copy-illegal,system_rhv_pem.unset,logs.open_source,keys.invalid,invalid_mac_ip,block_dev.small,no_space,inodes.less_inodes,env_leak
+            only option_root.single,invalid_rhv_pem,no-copy-illegal,system_rhv_pem.unset,logs.open_source,keys.invalid,invalid_mac_ip,block_dev.small,no_space,inodes.less_inodes
```
Comment (chunfuwen) [inline v2v/tests/cfg/function_test_esx.cfg:656]: maybe you can try  "no" keyword instead of only since this is already long string

Comment (xiaodwan) [reply v2v/tests/cfg/function_test_esx.cfg:656]: 'no' is unclear to users what cases are tested. And that still will be a long 'no' list.
I'd rather keep it.

**Comment by xiaodwan** [comment]
@yantian @chunfuwen Please help to review it, thanks.

**Comment by xiaodwan** [comment]
@yantian0 @chunfuwen Please help to review this PR, thanks.

**Review by yantian0 — APPROVED** [review]

**Pull request merged**